src/Generic/hooks/transfer-server.ts

```diff
@@ -58,14 +58,15 @@ export function useTransferInfos(
     domain => {
       return (transferInfosCache.get(domain) ||
         transferInfosCache.suspend(domain, async () => {
-          try {
-            const transferServer = await initTransferServer(domain, testnet)
-            if (!transferServer) return []
+          const transferServer = await initTransferServer(domain, testnet)
```

Comment:
Is it a good idea to move `initTransferServer()` out of the try-catch block? I think this call is just as likely to fail as the other one.

Reply:
That's what I thought at first but it actually does its own error handling (see [here](https://github.com/satoshipay/solar/blob/f330a29fae737ba23c89eea61c5cedb0061b96f0/src/Generic/hooks/transfer-server.ts#L23)). So it'd be redundant to also include it in the try-catch block.

Reply:
Good point, but i'd still keep it in the try-catch.

My thinking is that in this place in the code we don't know *how* `initTransferServer()` handles errors (unless you really look into the function's code you are calling). There is no guarantee that it will still catch any error tomorrow – maybe we change it to only catch certain errors that are anchor-related, for instance, and let others bubble up (like general errors that indicate an issue with our code). I doubt that whoever would change the `initTransferServer` function would also check every call to the function to make sure errors are handled properly by the caller.

WDYT?

Reply:
I see your point and I guess you're right that it is more robust that way so I'll change it back.